src/transaction.js

```diff
@@ -327,37 +408,92 @@ Transaction.prototype.byteLength = function () {
   return this.__byteLength(true)
 }
 
-Transaction.prototype.joinsplitByteLength = function () {
-  if (!this.overwintered && this.version < 2) {
+Transaction.prototype.getShieldedSpendByteLength = function () {
```

Comment:
Should this have the same network and version checks as `getJoinSplitByteLength`?